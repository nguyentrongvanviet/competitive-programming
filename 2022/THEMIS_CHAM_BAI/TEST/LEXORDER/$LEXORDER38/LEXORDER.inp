44 199224068728511523
40 36 43 10 7 22 25 30 16 17 3 42 32 13 19 21 8 2 1 6 31 23 4 20 15 11 41 37 27 12 5 24 38 34 39 9 14 28 33 35 18 26 29 44 