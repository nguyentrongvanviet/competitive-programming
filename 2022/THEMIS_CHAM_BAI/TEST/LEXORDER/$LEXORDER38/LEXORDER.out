40 36 43 10 7 22 25 30 16 17 3 42 32 13 19 21 8 2 1 6 31 23 4 20 18 38 5 41 24 9 33 37 34 27 15 14 35 12 29 39 11 28 44 26 